cmd/channel/distributed/dispatcher/main.go

```diff
@@ -50,18 +50,32 @@ import (
 var (
 	logger     *zap.Logger
 	dispatcher dispatch.Dispatcher
-	serverURL  = flag.String("server", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
-	kubeconfig = flag.String("kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 )
 
 // The Main Function (Go Command)
 func main() {
 
-	// Parse The Flags For Local Development Usage
-	flag.Parse()
+	ctx := signals.NewContext()
+
+	// Create The K8S Configuration (In-Cluster By Default / Cmd Line Flags For Out-Of-Cluster Usage)
+	k8sConfig := injection.ParseAndGetRESTConfigOrDie()
+
+	// TODO: do we really need these? I moved them up
```

Comment:
The QPS and burst: do we really need these? I moved them up since before the Kube client was created twice (once w/ QPS and burst, once w/o them) and now we create it once.

Reply:
> The QPS and burst: do we really need these?

I mean, it is totally fine to have these :) My main intention here is to understand why :)

Reply:
I'm not sure why these were put in place originally but since it's a constant setting of "1" times the default values of 5.0/10 for the QPS/Burst, and originally they were (re-)set after the BuildConfigFromFlags() function, I suspect it was more "ensuring that these values are the defaults" instead of anything more explicit.  I think setting the values to "0.0" and "0" with a comment that this forces the default values instead of loading them from the Kubernetes config would accomplish the same effect and possibly be less confusing.  Or maybe it would be more confusing :)

Reply:
Actually now that I look at it the code that sets the defaults for those two settings is in the ParseAndGetRESTConfigOrDie() function, so if the intention is to force the defaults the code likely needs to be the way it currently is.

Reply:
It looks like the code that translates "zero" into those defaults is in MainWithConfig->EnableInjectionOrDie, which is not called by this non-shared main.go function, so my suspicion is that this enforcing of the default values was taken from there and dropped here (meaning that this code should stay how it is, I think).